public_html/wp-content/plugins/camptix-attendee-survey/includes/admin-page.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Adds a page to the central admin.
+ */
+
+namespace CampTix\AttendeeSurvey\AdminPage;
+
+defined( 'WPINC' ) || die();
+
+use CampTix_Plugin;
+
+use function CampTix\AttendeeSurvey\{get_feature_id};
+use function CampTix\AttendeeSurvey\Email\{get_email_id};
+use function CampTix\AttendeeSurvey\Page\{get_page_id};
+use function CampTix\AttendeeSurvey\Cron\{get_wordcamp_attendees_id};
+
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Registered the appropriate filters and actions.
+ */
+function init() {
+	add_action( 'admin_menu', __NAMESPACE__ . '\admin_menu' );
+}
+
+/**
+ * Add a menu item.
+ */
+function admin_menu() {
+	add_menu_page(
+		__( 'WordCamp Attendee Survey', 'wordcamporg' ),
+		__( 'Attendee Survey', 'wordcamporg' ),
+		'manage_options',
+		get_feature_id(),
+		__NAMESPACE__ . '\render_menu_page',
+		'dashicons-feedback',
+		58
+	);
+}
+
+/**
+ * Get stats for each feedback for all sites.
+ *
+ * @return array[] An array of custom elements.
+ *                Each element has the following structure:
+ *                [
+ *                  'title' => string,
+ *                  'sent' => string,
+ *                  'responses' => string,
+ *                  'rate' => string,
+ *                ]
+ */
+function get_feedback_details() {
+	/* @var CampTix_Plugin $camptix */
+	global $camptix;
+
+	$wordcamps = get_posts( array(
+		'post_type' => WCPT_POST_TYPE_ID,
+		'post_status' => 'wcpt-closed',
+		'posts_per_page' => -1,
+	) );
+
+	$feedback_details = array();
+
+	foreach ( $wordcamps as $camp ) {
+		$site_id = get_post_meta( $camp->ID, '_site_id', true );
+
+		switch_to_blog( $site_id );
+
+		$blog_details = get_blog_details( $site_id );
+
+		// TODO: This should be tested elsewhere.
+		if ( 'events.wordpress.test' !== $blog_details->domain ) {
+			continue;
+		}
```

Comment:
I think this'll need to be updated to work on production.

You could use `get_top_level_domain()` in place of a hardcoded `.test`, but I think it'd be a bit more clear to check if `$site->blog_id === EVENTS_NETWORK_ID`.

Reply:
Yep. Makes sense.